test/mod/test_util_path.py

```diff
@@ -68,3 +68,41 @@ def test_in_tree():
 
     for args, expected in cases.items():
         assert path.in_tree(*args) == expected, args
+
+
+@pytest.mark.parametrize("test_case", (
+    {
+        "args": ("",),
+        "expected": "/"
+    },
+    {
+        "args": ("", "file"),
+        "expected": "/file"
+    },
+    {
+        "args": ("", "/file"),
+        "expected": "/file"
+    },
+    {
+        "args": ("/tmp", "/file"),
+        "expected": "/tmp/file"
+    },
+    {
+        "args": ("/tmp", "/foo", "/bar"),
+        "expected": "/tmp/foo/bar"
+    },
+    {
+        "args": ("/", "/foo"),
+        "expected": "/foo"
+    },
+    {
+        "args": ("/", "/foo", "/bar"),
+        "expected": "/foo/bar"
+    },
+    {
+        "args": (Path("/tmp"), "/foo", "/bar"),
```

Comment:
Should also test with Path objects in the paths list, both absolute and relative.

Also, should it test relative paths like `../foo`?

Reply:
+1 to the first comment.

For relative paths with dots, the `os.path.normpath()` will take care of those, but I can add a test case for it.

Reply:
I've added new test cases as a new commit.